config/_default/menus/menus.en.yaml

```diff
@@ -2609,20 +2609,22 @@ main:
     parent: cspm
     identifier: cspm_schema_gcp_test
     weight: 104
+  - name: Compliance Reports
+    weight: 105
```

Comment:
Are you leaving the URL as is? Also, are you leaving the Findings and Reports as a placeholder?